### DISCUSSION TITLE
Sorry for bragging but my osu won't download

### DISCUSSION THREAD

**Discussion by PureVannila** [ppy/osu#15803] [Q&A] [answered]
When downloading Osu! Lazer on my phone it won't work it says I need authorization I have a screenshot
(https://user-images.githubusercontent.com/95050379/143495449-cfbe443e-db26-482e-9466-6deb78927fac.jpg) as you can see

**Comment by bdach** [comment, marked as answer]
are you downloading the .apk from the supported location? namely https://github.com/ppy/osu#running-osu?

if so then i can assure you that the download works completely fine here and whatever problem you are running into is 99% on your end. it may be a question of device permissions, or your geographic location, or some configuration of the network you are using to download the file with.

**Reply by PureVannila** [reply]
Well I'm not but I'm trying to download it on android and the lazer version

**Reply by bdach** [reply]
where *are* you downloading the .apk from, then?

**Reply by PureVannila** [reply]
https://github.com/ppy/osu/releases from here

**Reply by MishterKirby** [reply]
Have you tried using an alternative browser?

**Reply by bdach** [reply]
then i can only restate what i said above. it works here, it must be something on your end. you will need to figure it out on your own.

i would recommend reviewing your router/network configuration or trying another network and seeing if the problem reproduces.

**Reply by PureVannila** [reply]
I haven't used a different browser and I will see if something is wrong with my network and stuff bc sometimes it does turn off by itself thank you both of you have a good day/night

**Reply by PureVannila** [reply]
a diffrent browser worked thanks
